commands/remove/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@lerna/remove",
+  "version": "3.10.6",
+  "description": "Remove dependency from matched packages",
+  "keywords": [
+    "lerna",
+    "command"
+  ],
+  "homepage": "https://github.com/lerna/lerna",
+  "license": "MIT",
+  "author": {
+    "name": "Daniel Stockman",
+    "url": "https://github.com/evocateur"
+  },
+  "files": [
+    "command.js",
+    "index.js",
+    "lib"
+  ],
+  "main": "index.js",
+  "engines": {
+    "node": ">= 6.9.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lerna/lerna.git"
+  },
+  "scripts": {
+    "test": "echo \"Run tests from root\" && exit 1"
+  },
+  "dependencies": {
+    "@lerna/command": "file:../../core/command",
+    "@lerna/filter-options": "file:../../core/filter-options",
+    "@lerna/npm-uninstall": "file:../../utils/npm-uninstall",
```

Comment:
`@lerna/npm-uninstall` looks like a copypaste of `@lerna/npm-install`. Why do we need it? We're already removing the entry in the package.json, etc. Running `lerna bootstrap` (like the add command does) should be sufficient to clean up the other stuff.

Reply:
not really, the bootstrap command will not update the lockfile, at least that's not happening in `yarn`, due to https://github.com/lerna/lerna/blob/master/commands/bootstrap/index.js#L171

Reply:
i guess that's the main motivation of the MR, because removing dep in `package.json` and running lerna bootstrap will not remove the dep from `node_modules` nor updating the lock file.

Reply:
Err, that branch is only reached when you are attempting a filtered bootstrap. (scope, ignore, or since)

The chained bootstrap should have none of these options, thus the raw bootstrap should actually deal with this?

Reply:
oh, my bad.
still, bootstrap will have early return if all the stated dependencies have been satisfied.
for our case (lerna remove), all stated dependencies is satisfied, but we have extraneous dependencies in node_modules, should we 
1. reuse bootstrap, but add a new flag, (something like https://github.com/lerna/lerna/issues/1883#issuecomment-459580755) to bypass the early return of lerna bootstrap, thus able to trigger npm client to remove extraneous pacakges and update lockfile or
2. use `npm-uninstall` that focuses on removing packages, by triggering npm client's `remove`/`uninstall` command rather than `install`/`ci` in `lerna bootstrap`

Reply:
Oh, right, I forgot Yarn doesn't want you to remove stuff when you modify `package.json` and run `yarn install`...

So yeah, I guess we do need `npm-uninstall`. I'd prefer it was less copypasta, but that's more work that isn't blocking.

We also, however, need to account for the `--hoist` case (the removed dependency doesn't exist at the leaf, but in the root) and relative `file:` specifiers, which need only re-run `npm install` in the root. `lerna bootstrap` already accounts for the `file:` specifier case, but I really don't want to burden that poor command with any _more_ complexity...

Reply:
For the `hoist` case, how should we figure out whether the dependency to be removed is hoisted or installed in leaf node?